src/static/dataloader.py

```diff
@@ -1,56 +1,183 @@
 import json
 import os
-
 import torch
 import xlrd
+from gdown import download
+
+# Google Drive file links for various models
+model_ids = {
```

Comment:
This dict has to go to a JSON file, which is downloaded in the beginning (the link of that file can be hardcoded, since no other will be later).